code-challenges.test.js

```diff
@@ -43,12 +98,48 @@ var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 
 // b) Create the function that makes the test pass.
 
-
+//Create a function called oddNumbers
+//  parameters - an array
+//  using filter, iterate through the array
+//      check the typeof the index
+//          if the typeof is a number, use the modulo to test for odd numbers and return the odd number
+//          else skip the value
+//  return the values sorted using sort().
+//END
+
+const oddNumbers = (array) => {
+    return array.filter(value => {
+        if(typeof value === 'number' && value %2 !== 0){
```

Comment:
This works! It's good to keep in mind that filter has a built in conditional statement. Otherwise we may want to use a for loop or some other kind of iteration.